package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.3",
-    "@adobe/helix-deploy": "1.17.3",
+    "@adobe/helix-deploy": "https://github.com/adobe/helix-deploy.git#no-epsagon",
```

Comment:
need to use a special version that doesn't include epsagon... need to make it configruable.